lib/N3Store.js

```diff
@@ -88,7 +89,9 @@ N3Store.prototype = {
   // (for instance: _subject_, _predicate_, and _object_).
   // Finally, `graph` will be the graph of the created triples.
   _findInIndex: function (index0, key0, key1, key2, name0, name1, name2, graph) {
-    var results = [], entityKeys = Object.keys(this._entities), tmp, index1, index2;
+    var vars = (key0 ? 1 : 0) + (key1 ? 1 : 0) + (key2 ? 1 : 0);
+    var results = [], entityKeys = (vars > 1 && !graph) ? Object.keys(this._entities) : this._entitiesInverse,
```

Comment:
`graph` is always truthy, so `Object.keys(this._entities)` never triggers.
(And that's for the better, because `this._entities['><'] = 0;` was never set, so it wouldn't work properly anyway.)